src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Services/OpenFileService.cs

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Utilities.UnifiedSettings;
+using Resx = NuGet.PackageManagement.VisualStudio.Strings;
+
+namespace NuGet.PackageManagement.VisualStudio.Services
+{
+    internal class OpenFileService : IArrayItemCommand
```

Comment:
This is more of a team-specific style issue, so feel free to ignore this feedback, but this doesn't feel like a "service" to me.  More like an "action".

Reply:
I agree. I just put it in the closest concept we had in this project. It could be perceived as a service that can open files in VS. I'm open to feedback on a new folder, etc.

Reply:
Are you opposed to calling it OpenFileAction even though it's in the Services namespace?